src/haz3lcore/statics/TermBase.re

```diff
@@ -89,7 +89,7 @@ and exp_term =
   | Test(exp_t)
   | Filter(stepper_filter_kind_t, exp_t)
   | Closure([@show.opaque] closure_environment_t, exp_t)
-  | Parens(exp_t) // (
+  | Parens(exp_t, Probe.tag)
```

Comment:
I think I'd vote for making an explicit Probe( ... ) form instead of using Parens. Parens to me feels like there should be no guarantee that they're preserved all the way through execution, and I can easily imagine someone writing a helper function that removes parens and creating unexpected bugs.